### PR TITLE
restore in order

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
restore in order.
@siddontang @iamxy PTAL

### CODE REVIEW

**Comment by iamxy** [comment]
LGTM

File: mydumper.c [hunk 1]
```diff
@@ -2621,9 +2621,9 @@ void dump_table(MYSQL *conn, char *database, char *table, struct configuration *
 			j->conf=conf;
 			j->type= is_innodb ? JOB_DUMP : JOB_DUMP_NON_INNODB;
 			if (daemon_mode)
-				tj->filename=g_strdup_printf("%s/%d/%s.%s.%05d.sql%s", output_directory, dump_number, database, table, nchunk,(compress_output?".gz":""));
+				tj->filename=g_strdup_printf("%s/%d/%s.%s.%09d.sql%s", output_directory, dump_number, database, table, nchunk,(compress_output?".gz":""));
```
Comment (siddontang) [inline mydumper.c]: why 09d here?

Comment (zhangjinpeng1987) [inline mydumper.c]: can store more files

**Comment by siddontang** [comment]
LGTM

File: mydumper.c (hunk 1 above)
Comment (morgo) [reply mydumper.c]: I just ran into this today -- was there a known use case where 5 digits was insufficient?

**Pull request merged**